src/domain/books/BooksPage.tsx

```diff
@@ -1,7 +1,20 @@
-export const BooksPage = (): JSX.Element => {
+import { BookCard } from "../../components/BookCard"
+import { Book, BooksState } from "./types"
+
+type BookPageProps = BooksState & {
+  handleAddToCart : (book:Book)=>void
+}
+
+
+export const BooksPage = ({loading,books,handleAddToCart}:BookPageProps): JSX.Element => {
   return (
-    <div>
-      Books Page
+    <div className="container">
+      {loading==="loading" && <h3>Loading</h3>}
+      {
+        books.map((book):JSX.Element=> 
+          <BookCard  book={book} handleAddToCart={handleAddToCart} />
+        )
+      }
```

Comment:
1st condition is okay. In the second block there should be a condition to check if loading is done and only then display the books